### PR TITLE
TCU airspace charts

### PR DESCRIPTION
**Pull request by alexd-332**
YWE
- Minor tweaks/fixes

ELW
- Minor tweaks/fixes

AY TWR
- Std assignable fixed

BIK
- Minor tweaks/fixes

Advanced Controller Skills
- Added maestro link

ML ADC
- Added TCU airspace division link

SY ADC
- Added TCU airspace division link

BN ADC
- Added TCU airspace division link

ML TCU
- Added TCU airspace division charts

SY TCU
- Added TCU airspace division charts

BN TCU
- Added TCU airspace division charts

CS TCU
- Created placeholder
- Added TCU airspace division charts

PH TCU
- Created placeholder
- Added TCU airspace division charts

TBD
- Created

ASP
- Created

TODO
- Updated

### CODE REVIEW

**Review by tylerthetiletiler — COMMENTED** [review]
File names need the spaces removed.

eg: docs/terminal/Brisbane TCU/img/01 PROPS annotated.png
needs to be docs/terminal/Brisbane TCU/img/01_PROPS_annotated.png

I'm also guessing the Cairns TCU files you've made are place holders?

**Comment by alexdiederich** [comment]
> File names need the spaces removed.
> 
> eg: docs/terminal/Brisbane TCU/img/01 PROPS annotated.png needs to be docs/terminal/Brisbane TCU/img/01_PROPS_annotated.png

fixed

> I'm also guessing the Cairns TCU files you've made are place holders?

affirm

**Review by tylerthetiletiler — APPROVED** [review]

**Review by mattkelly4 — DISMISSED** [review]

File: docs/enroute/Melbourne Centre/ASP.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+  title: Alice Springs (ASP)
+---
+
+--8<-- "includes/abbreviations.md"
+## Positions
```
Comment (chrisg06) [inline docs/enroute/Melbourne Centre/ASP.md:6]: Missing ESP in the positions list

Comment (alexdiederich) [reply docs/enroute/Melbourne Centre/ASP.md:6]: whoops, fixed

File: docs/terminal/Cairns TCU/coordination.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
+---
+  title: Coordination
```
Comment (chrisg06) [inline docs/terminal/Cairns TCU/coordination.md:2]: Page is a copy of ML TCU coordination

Comment (alexdiederich) [reply docs/terminal/Cairns TCU/coordination.md:2]: Yeah it's just a place holder, haven't done the actual page, just putting the TCU diagrams there

File: docs/terminal/Cairns TCU/operations.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+---
```
Comment (chrisg06) [inline docs/terminal/Cairns TCU/operations.md:1]: Copy of ML operations page

File: docs/terminal/Cairns TCU/coordination.md (hunk 2 above)
Comment (chrisg06) [reply docs/terminal/Cairns TCU/coordination.md:2]: Ah all good

**Review by chrisg06 — APPROVED** [review]
Looks good!

**Review by zachbf — APPROVED** [review]

**Pull request merged**